app/views/layout.mote

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head></head>
+    <body>
+
+
+
+        {{content}}
```

Comment:
Content is there, just index is a blank file unless you've loaded some peeps into the database.  If you look at the source you'll see that the <ul> tags from index is there